### PR TITLE
Update Maven to 3.6.3

### PR DESCRIPTION
**Pull request by colemarkham**

### CODE REVIEW

**Comment by kaylangan** [comment]
@leantk could you take a look at this?

**Comment by leantk** [comment]
LGTM

File: images/linux/Ubuntu1804-README.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ compiled using version 3.0.0.0 of the Cabal library )
 - Azul Zulu OpenJDK (12) (openjdk version "12.0.2" 2019-07-16)
 - Ant (Apache Ant(TM) version 1.10.5 compiled on March 28 2019)
 - Gradle 5.6.2
-- Maven (Apache Maven 3.6.1 (d66c9c0b3152b2e69ee9bac180bb8fcc8e6af555; 2019-04-04T19:00:29Z))
+- Maven (Apache Maven 3.6.3 (cecedd343002696d0abb50b32b541b8a6ba2883f))
```
Comment (kaylangan) [inline images/linux/Ubuntu1804-README.md]:
```suggestion
- Maven (Apache Maven 3.6.1 (d66c9c0b3152b2e69ee9bac180bb8fcc8e6af555; 2019-04-04T19:00:29Z))
```

Comment (kaylangan) [reply images/linux/Ubuntu1804-README.md]: We'll update the README once the change is deployed. The README is auto-generated.

**Comment by alepauly** [comment]
@colemarkham can you remove the change to the readme file (which should get rid of the conflict)? Once that's done I'll merge it in. Thanks!

**Comment by colemarkham** [comment]
@alepauly Fixed. I only updated the README because I noticed it was wrong. It says 3.6.1, but the version in the current image is 3.6.2.

**Comment by alepauly** [comment]
> 
> 
> @alepauly Fixed. I only updated the README because I noticed it was wrong. It says 3.6.1, but the version in the current image is 3.6.2.

Thanks for noticing it. It should become accurate soon (if it isn't already) as we go through finalizing the repo and making sure our processes are updating it correctly.

**Pull request merged**